types/node/index.d.ts

```diff
@@ -3217,6 +3270,13 @@ declare module "fs" {
         birthtime: Date;
     }
 
+    // TODO: Make this prettier
+    export interface Dirent extends Stats {
+    }
+    export class Dirent {
```

Comment:
Not sure if this solution to have classes and interfaces with same name is the best here. I'm not a big fan of copy paste but I think in this case it's easier to read. Other ideas are also welcome.

Reply:
This uses declaration merging, not sure if there is a better solution.

Reply:
This allows e.g. `Dirent.atime` but it should not.

Reply:
ew, fixed.